django-rgd-3d/rgd_3d/tasks/jobs.py

```diff
@@ -10,3 +10,13 @@ def task_read_mesh_3d_file(pc_file_pk):
 
     pc_file = Mesh3D.objects.get(pk=pc_file_pk)
     helpers._run_with_failure_reason(pc_file, read_mesh_3d_file, pc_file_pk)
+
+
+@shared_task(time_limit=86400)
```

Comment:
Minor thing, but I like to use `timedelta`s to express these sorts of values. Lots of libraries already accept them for time duration parameters, but even if they don't, you can just do:

```suggestion
@shared_task(time_limit=timedelta(days=1).total_seconds())
```

Which improves readability

Reply:
We should save this for a separate PR and change this everywhere... Perhaps by the introduction of a single setting/configuration?

Reply:
Good idea to make it a separate PR. In my other projects I do tend to make these config settings, but not necessarily because I want to tweak them all the time, more so just because it's a convenient place to dump tunable static values like this.